Jenkinsfile

```diff
@@ -1,70 +1,8 @@
 #!/usr/bin/env groovy
 
-REPOSITORY = 'search-admin'
-
 node {
   def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
-
-  properties([
-    buildDiscarder(
-      logRotator(
-        numToKeepStr: '50')
-      ),
-    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
-    [$class: 'ThrottleJobProperty',
-      categories: [],
-      limitOneJobWithMatchingParams: true,
-      maxConcurrentPerNode: 1,
-      maxConcurrentTotal: 0,
-      paramsToUseForLimit: 'search-admin',
-      throttleEnabled: true,
-      throttleOption: 'category'],
-  ])
-
-  try {
-    stage("Checkout") {
-      checkout scm
-    }
-
-    stage("Clean up workspace") {
-      govuk.cleanupGit()
-    }
-
-    stage("git merge") {
-      govuk.mergeMasterBranch()
-    }
-
-    stage("Configure Rails environment") {
-      govuk.setEnvar("RAILS_ENV", "test")
-    }
-
-    stage("bundle install") {
-      govuk.bundleApp()
-    }
-
-    stage("rubylinter") {
-      govuk.rubyLinter()
-    }
-
-    stage("Run tests") {
-      sh("bundle exec cucumber")
```

Comment:
Verified that we're still running cucumber tests: https://ci.integration.publishing.service.gov.uk/job/search-admin/job/update-ci/2/console